poglink/cogs/rates.py

```diff
@@ -19,43 +19,26 @@
 
 # create cog class
 class Rates(commands.Cog):
-    DEFAULT_SERVER_INFO = {
-        "smalltribes": {
-            "short_name": "Smalltribes",
-            "color": 0xA34C44,
-        },
-        "arkpocalypse": {
-            "short_name": "Arkpocalypse",
-            "color": 0xF8DE74,
-        },
-        "conquest": {
-            "short_name": "Conquest/Classic",
-            "color": 0x6DFF90,
-        },
-        None: {
-            "short_name": "Official",
-            "color": 0x63BCC3,
-        },
-    }
 
     def __init__(self, client):
         self.client = client
 
-        self.webpage_urls = client.config.rates_urls
+        self.rates_dicts = client.config.rates_urls
```

Comment:
Somewhat confusing nomenclature considering that it doesn't actually contain the rates themselves, but its the best I could come up with /shrug